style.css

```diff
@@ -15,14 +15,17 @@ h1 {
   background-repeat: no-repeat;
   background-size: cover;
   background-position: center;
+  position: relative;
 }
     
-.hero-text {
-  text-align: center;
+.hero-inner {
+  border: 1px solid pink;
+  width: 100 px;
+  height: 100px;
   position: absolute;
   top: 50%;
   left: 50%;
-  transform: translate(-50%, -50%);
+  margin: -125px 0 0 -125px;
```

Comment:
transform property is the correct solution, the margin is not. 

try to change values from negative too positive, it should work ok